gridcomps/History/MAPL_HistoryGridComp.F90

```diff
@@ -5395,15 +5416,12 @@ subroutine regen_rcx_for_obs_platform (config, nlist, list, rc)
        nseg_ub=0
        do while (ios == 0)
           read (unitr, '(A)' ) line
```

Comment:
I know it is not part of this pull request, but shouldn't there be an IOSTAT argument for the READ statement?   Seems robust, but I'm worried about trapping errors in running code.  I/O is the most common place for unexpected things to happen.

Reply:
Nice, that is a good point!  I see an example in the MAPL code: `Read(fIDGCHP,'(a)',IOStat=RC_) currLine`. I changed these similar lines in the code to follow the style to catch unexpected situations.